Components/InputTypeRadio/index.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export default function InputTypeRadio({
+  nameString,
+  valueString,
+  onChange,
+  idString,
+  labelString,
+}) {
+  function checked() {
+    nameString === valueString;
```

Comment:
It seems like you hardcoded the input for "nameString" and "valueString" in the LogScaleProperties/index.js.
The nameString can be either "logXAxis" or "logYAxis" and the valueString can be either "log" or "linear".
that mean they can never be equal and checked can never return true! (maybe its nice to return something here anyway :p

Please make sure this function works as expected :)

Reply:
It is true that the checked function will always return false. We have tried to use false as prop directly to the checked attribute in the return of InputTypeRadio component. However when we do that, then we don't see the blue dot in the radio button when we click on it.

On the other hand, when we use the checked function as we had written, then the code runs  correctly and the radio buttons are displayed correctly. Because of this we keep the code as we had it.